src/features/Dashboard/AssignedClasses/index.jsx

```diff
@@ -1,17 +1,22 @@
 import React, { useState, useEffect } from 'react';
 import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
 
 import { Col } from '@edx/paragon';
-import { Button } from 'react-paragon-topaz';
 
+import { useInstitutionIdQueryParam } from 'hooks';
 import ClassCard from 'features/Dashboard/AssignedClasses/ClassCard';
 
 import 'features/Dashboard/AssignedClasses/index.scss';
 
+const displayClassLimit = 3;
```

Comment:
@01001110J what do think about adding this value in constants.js?

Reply:
@sergivalero20  We should evaluate if we need it in other page to make sense to move to constants, in this case, the scope of this variable is only this component so It's ok to leave it in this page

Reply:
I consider a good idea to move it into a constant @AuraAlba would be useful for [PADV-1571](https://agile-jira.pearson.com/secure/RapidBoard.jspa?rapidView=51807&view=detail&selectedIssue=PADV-1571#)

Reply:
In what case you will be use that constant for that story? @01001110J